skins/Belchertown/index.html.tmpl

```diff
@@ -20,137 +20,147 @@
+                if ( ( Math.floor(new Date().getTime()/1000.0) - $Extras.last_updated_alert_threshold ) > $current.dateTime.raw ) {
+                    belchertown_debug("Last Updated Alert: The weewx report timestamp is "+$current.dateTime.raw+". The current timestamp is "+Math.floor(new Date().getTime()/1000.0)+". The threshold is "+$Extras.last_updated_alert_threshold+". Report timestamp is greater than threshold of current time, showing no update alert.");
+                    last_updated_alert_text = "<i class='fa fa-exclamation-triangle' style='color:red;'></i> $obs.label.header_last_updated_alert";
+                    jQuery(".last-updated-alert").html( last_updated_alert_text );
+                    jQuery(".last-updated-alert").show();
+                }
+                #end if
                         
-            // Update the Last Updated timestamp with moment.js,
-            updated = moment.unix( $current.dateTime.raw ).utcOffset($moment_js_utc_offset).format("$obs.label.time_last_updated");
-            updated_text = "$obs.label.header_last_updated " + updated;
-            jQuery(".updated").html( updated_text );
+                // Update the Last Updated timestamp with moment.js,
+                updated = moment.unix( $current.dateTime.raw ).utcOffset($moment_js_utc_offset).format("$obs.label.time_last_updated");
+                updated_text = "$obs.label.header_last_updated " + updated;
+                jQuery(".updated").html( updated_text );
             
-            #end if
+                #end if
             
-            //  display the relevant forecast; a new session will display the default forecast, a reloaded session the currently displayed forecast
-            if ( sessionStorage.getItem ("forecastInterval")  ) {
-                interval = sessionStorage.getItem("forecastInterval");
-                sessionStorage.setItem ( "forecastInterval" , null );
-                forecast_select( interval );
-            } else {
-                sessionStorage.setItem ( "forecastInterval" , null );
-                forecast_select( sessionStorage.getItem( "defaultInterval") );
-            }
+                //  display the relevant forecast; a new session will display the default forecast, a reloaded session the currently displayed forecast
+                if ( sessionStorage.getItem ("forecastInterval")  ) {
+                    interval = sessionStorage.getItem("forecastInterval");
+                    sessionStorage.setItem ( "forecastInterval" , null );
+                    forecast_select( interval );
+                } else {
+                    sessionStorage.setItem ( "forecastInterval" , null );
+                    forecast_select( sessionStorage.getItem( "defaultInterval") );
+                }
+            });
+        }).catch(function(e) {
+            console.log(e);
         });
         
     </script>
```

Comment:
I'm confused why this is all within this PR? I think this code is fine? Where is the change?